### PR TITLE
Update python environment/setup

### PR DESCRIPTION
**Pull request by swedchef**
The main .py file has been renamed to more correct name and the
python requirements have been moved to requirements.txt. The
Dockerfile has been updated to reflect the changes.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,10 +1,12 @@
-# The base image
-# Load python container and install required python libs
+# Select base-image
 FROM python:3.10.0-slim-bullseye
-RUN pip3 install kafka-python pandas requests --no-cache-dir
 
-# Cpy required files into container
-COPY forecast-import.py ksql-config.json gridpoints.csv /
+# Load python requirements, install them and clean up.
+COPY requirements.txt requirements.txt
```
Comment (stigsberg) [inline Dockerfile:5]: The syntax in line 5 looks wierd to me

**Comment by stigsberg** [comment]
Only have 1 comment otherwise it LGTM

File: Dockerfile (hunk 1 above)
Comment (swedchef) [reply Dockerfile]: Fixed so it looks identical to the other - should work no matter what though! ;)

**Review by CodeWitten — APPROVED** [review]
så fint

**Pull request merged**